### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -15,14 +15,13 @@ source /usr/share/yunohost/helpers
 ynh_script_progression --message="Setting up source files..." --weight=4
 
 # Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source --dest_dir=$install_dir --source_id=$YNH_ARCH
+ynh_setup_source --dest_dir=$install_dir
 
 pushd "$install_dir"
 	gzip --decompress $YNH_ARCH.gz
 	mv $YNH_ARCH $app
```
Comment (alexAubin) [inline scripts/install:22]:
```suggestion
	gzip --decompress main.gz
	mv main $app
```

With the new sources thing, it's probably gonna be named "main" (same thing during upgrade)

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 2]
```diff
@@ -15,14 +15,13 @@ source /usr/share/yunohost/helpers
...
 
 # Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source --dest_dir=$install_dir --source_id=$YNH_ARCH
+ynh_setup_source --dest_dir=$install_dir
 
 pushd "$install_dir"
-	gzip --decompress $YNH_ARCH.gz
-	mv $YNH_ARCH $app
+	gzip --decompress main.gz
+	mv main $app
```
Comment (alexAubin) [inline scripts/install:22]:
```suggestion
	gzip --decompress main --to-stdout > main_dezipped
	mv main_dezipped $app
```

**Pull request merged**